lib/typhoeus/request.rb

```diff
@@ -176,6 +176,14 @@ def encoded_body
       Ethon::Easy::Form.new(nil, options[:body]).to_s
     end
 
+    def to_s
+      <<STR
+#{self.class}
+  url: #{url}
+  options: #{options}
+STR
+    end
+
     private
```

Comment:
@iamvery what do you think about this approach, removing the multiline heredoc, just to keep sane alignment (and btw, note you could use ````<<-STR```` , note dash,  for a more flexible closing token check) and adding basic documentation:

````ruby
    # Return a string representation of the object
    #
    def to_s
      "#{self.class}(url: #{url}, options: #{original_options})"
    end
````